### ISSUE TITLE
Interface lagging with very high resolution images

### ISSUE THREAD

**Issue by montmejat** [SkalskiP/make-sense#292]
There are two issues that make labeling very high resolution images complicated (and especially for labeling small objects):

 * The scrolling direction is not controlled by the mouse.
 * The scrolling and moving when zoomed in is laggy.

You can find an example, with a video of what it looks like compared to Label Studio (which would be nice to have something similar) here: https://github.com/SkalskiP/make-sense/issues/289#issuecomment-1293226529.

**Comment by ajansenn** [comment]
Would also find this improvement valuable. I’m trying to label high resolution drone images (8500x6500 dimensions and ~30mb per image) with up to 1000 polygons per image. Haven’t found an annotation tool that doesn’t lag or eventually crash.

**Comment by SkalskiP** [comment]
Hi, @ajansenn! 👋 That sounds like a massive project. I'm not sure anyone can provide you with an in-browser smooth experience. Have you considered cutting those images into smaller sections and annotating them separately?

**Comment by rasyidf** [comment]
In browser smooth experience will require technology like WebGL or WASM for rendering purpose. while we can decrease the lag with Webworker, it will require some tweaks. I agree this will be a huge task.

**Comment by SkalskiP** [comment]
I agree with @rasyidf. WASM could solve the problem. That's how Photoshop in the browser works - obviously, a much more complicated and resource-hungry app. But it would require a ton of work. For that reason, I don't think we will be able to help @ajansenn. However, I think we will certainly be able to solve @aurelien-m problems.
